plugins/modules/packaging/os/homebrew_tap.py

```diff
@@ -49,7 +49,7 @@
     path:
         description:
             - "A ':' separated list of paths to search for C(brew) executable."
```

Comment:
While we're at it ;)
```suggestion
            - "A C(:) separated list of paths to search for C(brew) executable."
```